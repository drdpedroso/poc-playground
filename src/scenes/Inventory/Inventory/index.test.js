import React from 'react'
import { Inventory } from './index'
import renderer from 'react-test-renderer'

describe('Inventory ', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Inventory />)
    expect(component).toMatchSnapshot()
  })
})
