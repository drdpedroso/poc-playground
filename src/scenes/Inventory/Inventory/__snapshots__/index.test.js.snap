// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inventory  should render correctly 1`] = `
<div>
  Leke2222
</div>
`;
